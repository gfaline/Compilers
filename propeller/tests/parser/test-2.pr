# A simple function that's syntactically valid, but semantically meaningless
fn first_function () -> str
{
  str a;
  a = HelloWorld;
  return a;
} 