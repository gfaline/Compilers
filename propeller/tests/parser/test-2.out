
fn first_function() -> str
{
str a;
a = HelloWorld;
return a;
}